DatabaseAndReporting/Operation/01_ExecuteManagedQueries.md

```diff
@@ -0,0 +1,140 @@
+[!!Manage the queries](../Integration/01_ManageQueries.md)
+[!!Manage the query categories](../Integration/02_ManageQueryCategories.md)
+[!!Manage the user rights](../Integration/05_ManageUserRights.md)
+[!!User interface Managed queries](../UserInterface/01a_Queries.md)
+
+# Execute the managed queries
+
+Once the queries have been predefined, they can be executed. When executing a query, the query results can be provided in CSV or XML format, either as a download file or as a preview.
+
+Depending on the options that have been enabled in the query, a query can be executed via API or via download link. Moreover, the managed queries may only be executed by users who have been granted the applicable access rights. For detailed information, see [Manage the user rights](../Integration/05_ManageUserRights.md). 
+
+
+
+## Execute a query via API 
+
+Execute a predefined query to access the available data in your Core1 account. To execute a query via API, the user must be logged in. The query results may be previewed in your browser in CSV or XML format or may be downloaded in a CSV or an XML file. 
+
+
+### Preview the query results
+
+Execute the query and preview the results in your browser in CSV or XML format.
+
+#### Prerequisites
+
+- A predefined query has been created, see [Create a query](../Integration/01_ManageQueries.md#create-a-query).
+- The API and download option has been enabled. 
+- The user has been granted the applicable access rights, see [Grant access rights to a user group](../Integration/05_ManageUserRights.md#grant-access-rights-to-a-user-group).
+
+#### Procedure
+
+*Database and reporting > Managed queries > Tab QUERIES*
+
+![Queries](../../Assets/Screenshots/DatabaseAndReporting/ManagedQueries/Queries/Queries.png "[Queries]")
+
+1. Select the checkbox of the query you want to preview.  
+    The editing toolbar is displayed.
+
+2. Click the file format drop-down list in the editing toolbar and select the desired option. The following options are available:  
+      - **CSV**   
+        The query results are displayed in CSV format.   
+      - **XML**   
+        The query results are displayed in XML format.  
+
+3. Click the ![Preview](../../Assets/Icons/Eye02.png "[Preview]") (Preview) button in the editing toolbar.  
+    The *Execute saved query* is displayed.  
+
+    ![Execute saved query](../../Assets/Screenshots/DatabaseAndReporting/ManagedQueries/Queries/ExecuteSavedQuery.png "[Execute saved query]")
+
+4. If the selected query contains a placeholder, the *Bind query parameters placeholders* window is displayed. Otherwise, go to step **6**.
```

Comment:
```suggestion
4. If the selected query contains a placeholder, for example a creation date field, the *Bind query parameters placeholders* window is displayed. Otherwise, go to step **6**.
```